data_model/src/isi.rs

```diff
@@ -869,6 +872,16 @@ pub mod error {
             #[display(fmt = "This asset was set as infinitely mintable. You cannot forbid its minting.")]
             ForbidMintOnMintable,
         }
+
+        /// Invalid instruction parameter error
+        #[derive(Debug, Display, Clone, PartialEq, Eq)]
+        #[ffi_type(opaque)]
+        #[repr(u8)]
+        pub enum InvalidParameterError {
+            /// Invalid WASM binary
+            #[display(fmt = "Invalid WASM binary: {_0}")]
+            Wasm(String),
+        }
```

Comment:
Why is this enum-error uses `derive_more::Display` and not `thiserror::Error`? I though the latter is designed specifically to handle enum-errors.

Reply:
because in `no_std` environments you cannot derive `thiserror::Error`

Reply:
@0x009922 , can we close this?